.env

```
@@ -1,10 +1,10 @@
+APP_NAME=hyku
```

We set this to the current directory when using stack_car. This ensures if you have multiple Hyku instances running they all get their own name. That's why it wasn't set before. However, I did some experimentation and it looks like we can do it this way and have it work in both cases. Does that work for you?

```suggestion
APP_NAME=${APP_NAME:-hyku}
```

---

Totally - let me try it in my local environment and I'll update the PR if it works.

---

@orangewolf I'm not sure I understand the functional difference between the the original and your proposed change. If APP_NAME is set in a parent process (e.g my shell or stack_car), won't that take precedence over the value set in .env?

Here's my test:
If I run with the initial .env formulation, e.g. `APP_NAME=hyku`
```
APP_NAME=my-dir docker compose up
```
The app is accessible at https://admin-my-dir.localhost.direct/ and I get
```
irb(main):001> ENV['HYKU_ADMIN_HOST']
=> "admin-my-dir.localhost.direct"
```

if I run with the proposed change, e.g. `APP_NAME=${APP_NAME:-hyku}`
```
APP_NAME=my-dir docker compose up
```
The app is accessible at https://admin-my-dir.localhost.direct/ and I get
```
irb(main):001> ENV['HYKU_ADMIN_HOST']
=> "admin-my-dir.localhost.direct"
```

They look the same to me, so I'd like the simpler formulation in the .env file if possible. But if I'm missing something about how stack_car interacts with the stack overall, I'm happy to put the change in.